ci/test-container.sh

```diff
@@ -15,9 +15,9 @@ case $versionid in
     ignition_url_suffix=2.16.2/2.fc39/x86_64/ignition-2.16.2-2.fc39.x86_64.rpm
     # 2.15.0-3
     koji_ignition_url="https://koji.fedoraproject.org/koji/buildinfo?buildID=2158585"
-    koji_kernel_url="https://koji.fedoraproject.org/koji/buildinfo?buildID=2435097"
-    kver=6.8.5
-    krev=300
+    koji_kernel_url="https://koji.fedoraproject.org/koji/buildinfo?buildID=2496213"
+    kver=6.9.10
+    krev=200
```

Comment:
We usually use the kernel package in the stable repo which shouldn't be GC'ed. In this case it's:

```
$ koji latest-pkg f40 kernel
Build                                     Tag                   Built by
----------------------------------------  --------------------  ----------------
kernel-6.8.5-301.fc40                     f40                   jforbes
```

--> https://koji.fedoraproject.org/koji/buildinfo?buildID=2436096.

Reply:
thank you!